solutions/Hawk/org.hawk.ttc2018/src/org/hawk/ttc2018/AbstractLauncher.java

```diff
@@ -99,11 +99,11 @@ public PhaseWrapper(int iteration, Phase phase) {
 
 		public void close() {
 			final long elapsedTime = System.nanoTime() - startNanos;
-			final long availableBytes = Runtime.getRuntime().totalMemory() - Runtime.getRuntime().freeMemory();
+//			final long availableBytes = Runtime.getRuntime().totalMemory() - Runtime.getRuntime().freeMemory();
 
 			new Snapshot(iteration, phase, Metric.Time, elapsedTime).print(System.out);
-			System.gc();
-			new Snapshot(iteration, phase, Metric.Memory, availableBytes).print(System.out);
+//			System.gc();
+//			new Snapshot(iteration, phase, Metric.Memory, availableBytes).print(System.out);
```

Comment:
@bluezio
Although I've commented this out, but we should have a look at this.
Is this correct? It looks like the RAM usage is first measured then the GC is invoked. Should this be in reverse order?